web/components/Card.js

```
@@ -0,0 +1,264 @@
+        .from("fireplace-videos")
+        .delete()
+        .eq("id", id);
+
+      if (err || error) {
+        reject();
+        throw new Error(err);
+      } else {
+        setVideos(prevVideos => prevVideos.filter((video) => video.id != id));
+        resolve();
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)